src/libtriton/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ endif()
 
 # Default flags
 # To fix old python version bug http://bugs.python.org/issue21958
-if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows" AND NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DHAVE_ROUND /MT")
 endif()
 
```

Comment:
Quoted variables, such as  "${CMAKE_CXX_COMPILER_ID}" , will cause a waring in windows CMake  version 3.7.2. 
`CMake_ Warning (dev) at src/libtriton/CMakeLists.txt:21 (if):`
`  Policy CMP0054 is not set: Only interpret if() arguments as variables or keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy details.  Use the cmake_policy command to set the policy and suppress this warning.`

`  Quoted variables like "MSVC" will no longer be dereferenced when the policy is set to NEW.  Since the policy is not set the OLD behavior will be used.`

Line 11 add  -D_CRT_SECURE_NO_WARNINGS , to avoid a few waring, such as:
`abstractBinary.cpp(50): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.`